shapeshift/web
# feat: turn on CowSwap flag

## Description

CowSwap fees are now correctly sent from `lib`, and consumed by `swapperv2`.

We should be ok to turn this on.

## Notice


## Pull Request Type


## Issue (if applicable)

Contributes to https://github.com/shapeshift/web/issues/2873

## Risk

Medium - routes all trades that can be handled via CowSwap via CowSwap.

## Testing

- Ensure CowSwap trades and approvals work as expected
- Ensure fees are shown as expected on both the quote screen and confirm screen, and that they match the amount actually received when executing the trade.

### Engineering

As above.

### Operations

As above.

## Screenshots (if applicable)

N/A
